### ISSUE TITLE
Screen reader clashing with windows mode

### ISSUE THREAD

**Issue by vyncent-t** [kubernetes-sigs/headlamp#4366]
## Describe the bug

When using a screen reader, the resource list table in Headlamp is announced correctly. However, when a resource is opened in drawer view mode, the underlying table **remains focusable.** 

This causes the screen reader to move focus and announce content behind the open drawer. Focus should be trapped within the drawer to prevent background interaction and confusion.

Notice that the tab focus is highlighting the Ready section underneath:

<img width="1607" height="227" alt="Image" src="https://github.com/user-attachments/assets/c34e8f81-9134-4b1f-9a11-f6b7eebaec00" />

## To Reproduce
Steps to reproduce the bug:
Be sure you are using Headlamp in window (drawer) mode, switch to this in your settings first.

1. Open workloads and navigatge to pods
2. Open any pod resource in window view
3. Activate screen reader and start tab navigation
4. Notice the navigation and focus narration occurs ontop of the drawer window

## Are you able to fix this issue?
Will be looking into

**Comment by vyncent-t** [comment]
Images:

The namespace UI is interactable from an open drawer

<img width="1603" height="249" alt="Image" src="https://github.com/user-attachments/assets/1093e07e-a3fa-476a-a698-33d702c7c8e1" />

**Comment by Sagar-6203620715** [comment]
Hi @vyncent-t Would you mind if I do this if you have not started yet.

**Comment by vyncent-t** [comment]
> Hi [@vyncent-t](https://github.com/vyncent-t) Would you mind if I do this if you have not started yet.

Sure thing 👍 @Sagar-6203620715
